### ISSUE TITLE
I missed Zeus Arcana From Candy's Shop when buy it

### ISSUE THREAD

**Issue by Minoyo97** [ValveSoftware/Dota2-Gameplay#16990]
I buy Zeus Arcana in Candy Store/ Then game crashed / When I back I haven't seen Arcana for Zeus
![Снимок экрана 2024-05-01 165906](https://github.com/ValveSoftware/Dota2-Gameplay/assets/168652131/083e430e-6f69-4135-a3ff-2990a8b93d88)
![Снимок экрана 2024-05-01 1700021](https://github.com/ValveSoftware/Dota2-Gameplay/assets/168652131/e80548be-06d0-4d8f-8a1b-df6983390214)
![Снимок экрана 2024-05-01 165906](https://github.com/ValveSoftware/Dota2-Gameplay/assets/168652131/8b926894-77e4-4ec7-b04f-ce32148524da)
![Снимок экрана 2024-05-01 1700021](https://github.com/ValveSoft
![123414214124](https://github.com/ValveSoftware/Dota2-Gameplay/assets/168652131/532b6b69-6165-4210-a0df-68c2ad0e987d)
ware/Dota2-Gameplay/assets/168652131/1e59092a-d2ae-48c9-ad86-bdb3169c1a82)

**Comment by DEEEEFAit** [comment]
it needs to be unpacked through the inventory or the button on the top right

**Comment by Minoyo97** [comment]
Nope. I haven't it in the button on the top right. It dissapear everywhere/ On this button/ In the inventory Dota/ In the inventory steam

**Comment by B1st777** [comment]
> I buy Zeus Arcana in Candy Store/ Then game crashed / When I back I haven't seen Arcana for Zeus
> ![Снимок экрана 2024-05-01 165906](https://github.com/ValveSoftware/Dota2-Gameplay/assets/168652131/083e430e-6f69-4135-a3ff-2990a8b93d88)
> ![Снимок экрана 2024-05-01 1700021](https://github.com/ValveSoftware/Dota2-Gameplay/assets/168652131/e80548be-06d0-4d8f-8a1b-df6983390214)
> ![Снимок экрана 2024-05-01 165906](https://github.com/ValveSoftware/Dota2-Gameplay/assets/168652131/8b926894-77e4-4ec7-b04f-ce32148524da)
> ![Снимок экрана 2024-05-01 1700021](https://github.com/ValveSoft
> ![123414214124](https://github.com/ValveSoftware/Dota2-Gameplay/assets/168652131/532b6b69-6165-4210-a0df-68c2ad0e987d)
> ware/Dota2-Gameplay/assets/168652131/1e59092a-d2ae-48c9-ad86-bdb3169c1a82)
> 

Bro, you will get it, when servers will be ok, keep calm & carry on!

**Comment by Eysti** [comment]
Same. I got only head slot from the arcana set and the game thinks that I don't own the arcana :c

**Comment by Eysti** [comment]
![Uploading изображение_2024-05-02_020812269.png…]()

**Comment by Eysti** [comment]
![Uploading ggggggggggg.png…]()

**Comment by Eysti** [comment]
![ggggggggggg](https://github.com/ValveSoftware/Dota2-Gameplay/assets/70438280/236f87d1-da14-4142-8951-028bad0ae891)
